Shuttlerock/actions
# [STUDIO-603] Handle users with private Jira email addresses

## Handle users with private Jira email addresses

[Jira Tech task](https://shuttlerock.atlassian.net/browse/STUDIO-603)



## How to test the PR

Make your email address private in Jira, and add your Jira user name in https://next.shuttlerock.com/integrations/edit . When a PR is created, it should look up the user by Jira name instead of email, if email is not available.

## Deployment Notes

None.
